package.json

```diff
@@ -50,7 +50,9 @@
     "node-mocks-http": "1.2.3",
     "sinon": "1.17.6",
     "typescript": "2.0.10",
-    "typings": "2.0.0"
+    "@types/mocha": "^2.2.40",
+    "@types/node": "^7.0.12",
```

Comment:
Going down to 4.x should be fine if that's the earliest available. Our CI builds will verify things still run on the older versions